components/Relationships/RelationshipElement.tsx

```diff
@@ -2,35 +2,118 @@ import Link from 'next/link';
 import { RelationshipPerson } from 'types';
 
 interface Props {
-  title: string;
-  data: RelationshipPerson[];
+  type: string;
+  persons: RelationshipPerson[];
```

Comment:
`people`?

Reply:
that is what comes from the backend response :)